database/schema.sql

```diff
@@ -1,4 +1,210 @@
-CREATE TABLE example_table (
-    created_at timestamp ,
-    id SERIAL PRIMARY KEY
+create table r12Elector ( 
```

Comment:
In postgres, if you don't add quotes around table and column names it will automatically downcase them.

Reply:
The tables in the spike document have `dbo` in front of them.

Should we add `CREATE SCHEMA IF NOT EXISTS dbo;
` and the `dbo.` prefix to the tables here?